packages/patterns/src/convertToCSS.ts

```diff
@@ -14,13 +14,18 @@
  * limitations under the License.
  */
 
-function convertToCSS(style: Record<string, string>) {
+/**
+ * External dependencies
+ */
+import type { CSSProperties } from 'react';
+
+function convertToCSS(style: CSSProperties) {
   return Object.entries(style).reduce((str, [key, val]) => {
```

Comment:
Could also add the type `string` here for `val` instead, maybe it's better than casting?

Reply:
I wasn't happy about this, but didn't see a better way. I'd have to type the whole array here, so it would become:

```ts
  return Object.entries(style).reduce((str, [key, val]: [string, string]) => {
```

Which doesn't look that good IMO.

Reply:
I've changed it to:

```
return `${str}${casedKey}:${String(val)};`;
```

That looks at least a bit better. And would correctly convert numbers (which are the most likely non-strings here).